src/network/command.rs

```diff
@@ -136,7 +136,7 @@ impl CommandMessage {
         bytes[0x21] = 0;
 
         // Ensure that the checksums match
-        let real_checksum = checksum(&bytes);
+        let mut real_checksum = checksum(&bytes);
```

Comment:
Could you leave this as immutable and then shadow the variable later? I think that is more idiomatic since it shows that it doesn't change until a later rebind.

Reply:
You're right. I've adjusted the PR.